conda/plugins/manager.py

```diff
@@ -50,19 +54,38 @@ def load_plugins(self, *plugins) -> list[str]:
                 plugin_names.append(plugin_name)
         return plugin_names
 
-    def load_setuptools_entrypoints(self, *args, **kwargs) -> int:
-        """
-        Overloading the parent method from pluggy to add conda specific exceptions.
-
-        See :meth:`pluggy.PluginManager.load_setuptools_entrypoints` for
-        more information.
+    def load_entrypoints(
+        self, group: str, name: str | None = None
+    ) -> int:
+        """Load modules from querying the specified setuptools ``group``.
+        :param str group: Entry point group to load plugins.
+        :param str name: If given, loads only plugins with the given ``name``.
+        :rtype: int
+        :return: The number of plugins loaded by this call.
         """
-        try:
-            return super().load_setuptools_entrypoints(*args, **kwargs)
-        except Exception as err:
-            raise PluginError(
-                f"Error while loading conda plugins from entrypoints: {err}"
-            )
+        count = 0
+        for dist in list(distributions()):
+            for entry_point in dist.entry_points:
+                if (
+                    entry_point.group != group
+                    or (name is not None and entry_point.name != name)
+                    # already registered
+                    or self.get_plugin(entry_point.name)
+                    or self.is_blocked(entry_point.name)
+                ):
+                    continue
+                try:
+                    plugin = entry_point.load()
+                except Exception as err:
+                    # not using exc_info=True here since the CLI loggers are
+                    # set up after CLI initialization and argument parsing,
+                    # meaning that it comes too late to properly render
+                    # a traceback
+                    log.warning(f"Could not load conda plugin `{entry_point.name}`:\n\n{err}")
+                    continue
+                self.register(plugin, name=entry_point.name)
+                count += 1
+        return count
```

Comment:
How do we get this line to run

Reply:
Not following, this just returns the count, there is no return statement in the previous code. This contains code from https://github.com/pytest-dev/pluggy/blob/c18d5da87f33bc5359c6eeccf4e85a194842d907/src/pluggy/_manager.py#L352-L377 BTW